keyboards/dz60/keymaps/iso_br_abnt2/keymap.c

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2019 Luiz Correia
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_br_abnt2.h"
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * _______ can be used in place of KC_TRNS (transparent) *
+ * XXXXXXX can be used in place of KC_NO (No Operation)  *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#define _BL 0
+#define _FL 1
+#define _CL 2
```

Comment:
Optional, but recommended: use an `enum` to manage the layers:

```c
enum layer_names {
    _BL,
    _FL,
    _CL,
};
```

This is the same as the set of definitions you have currently once compiled.